build/Dockerfile

```diff
@@ -1,21 +1,10 @@
-FROM registry.access.redhat.com/ubi7/ubi-minimal:latest
+FROM scratch
 
-RUN rpm -e pinentry gnupg2 gpgme librepo libdnf microdnf pth \
-           shared-mime-info glib2 json-glib gobject-introspection libpeas \
-           librhsm pkgconfig libmodulemd libmount expat libsolv libyaml \
-           libsmartcols json-c libgcrypt libxml2 libblkid libassuan libuuid \
-           libgpg-error
-
-ENV OPERATOR=/usr/local/bin/submariner-operator \
```

Comment:
We don't need those env vars anymore?

Reply:
No, there were used by the startup scripts, which we no longer need. (In fact I need to delete them...)